e2e/test/provisioning/ProvisioningE2ETests.cs

```diff
@@ -896,44 +896,34 @@ public static ProvisioningClientOptions CreateProvisioningClientOptionsFromName(
         private static async Task ConfirmRegisteredDeviceWorksAsync(
             DeviceRegistrationResult result,
             IAuthenticationMethod auth,
-            IotHubClientTransportSettings transportSettings,
-            bool sendReportedPropertiesUpdate)
+            IotHubClientTransportSettings transportSettings)
         {
             await using var iotClient = new IotHubDeviceClient(result.AssignedHub, auth, new IotHubClientOptions(transportSettings));
```

Comment:
can we rename `iotClient` to something else, like `deviceClient`?

Reply:
I'll change it, but in a different PR unless I need to restart gates.